python/dcr_template.py

```diff
@@ -962,40 +1048,35 @@ def __init__(self, repository=".", obsid_range=None, band_name='g', wavelength_s
         else:
             self.exposures = exposures
 
+        if len(self.exposures) == 0:
+            raise ValueError("No valid exposures found.")
+
+        self.debug = debug_mode
+        self.instrument = instrument
         self.n_images = len(self.exposures)
+        self.detected_bit = detected_bit
+        self.filter_name = band_name
         psf_size_arr = np.zeros(self.n_images)
```

Comment:
The psf size was originally set to the smallest back when I was using the linear least squares approach and the size of the psf kernel greatly impacted memory use and execution time. That's no longer a major factor, so I've changed it to use the maximum and added a comment.